nifi-docs/src/main/asciidoc/developer-guide.adoc

```diff
@@ -1751,6 +1751,12 @@ will handle your Processor:
 		will always be set to `1`. This does *not*, however, mean that the Processor does not have to be thread-safe,
 		as the thread that is executing `onTrigger` may change between invocations.
 
+    - `PrimaryNodeOnly`: Apache NiFi, when clustered, offers two modes of execution for Processors: "Primary Node" and
+        "All Nodes". Although running in all the nodes offers better parallelism, some Processors are known to cause unintended
+        behaviors when run in multiple nodes. For instance, some Processors lists or reads files from remote filesystems. If such
```

Comment:
Typo in the docs: think it should read "some Processors list or read files" rather than "lists of reads"

Reply:
Fixed